core/services/relay/types/relay.go

```diff
@@ -13,32 +13,55 @@ import (
 type Network string
```

Comment:
Consider moving this file up a level, and we could rename the `Relayer` as `Service` so we'd type it as:

- `relay.PluginArgs`
- `relay.ConfigWatcherArgs`
- `relay.Service`

Reply:
I'm gonna follow up by pulling this into chainlink-relay, will do then